server/src/test/java/org/apache/iotdb/db/writelog/WriteLogNodeTest.java

```diff
@@ -180,9 +180,13 @@ public void testSyncThreshold() throws IOException, IllegalPathException {
 
     File walFile = new File(
         config.getWalDir() + File.separator + "root.logTestDevice" + File.separator + "wal1");
-    assertTrue(!walFile.exists());
+    assertFalse(walFile.exists());
 
     logNode.write(deletePlan);
+    System.out.println("Waiting for wal file to be created");
+    while (!walFile.exists()) {
+
```

Comment:
better to print something after a period.

Reply:
Unnecessary, CI will do the job.